test/image.bats

```diff
@@ -293,3 +294,30 @@ function teardown() {
 
 	cleanup_images
 }
+
+@test "run container in pod with crun-wasm enabled" {
+	if [ -z "$CRUN_WASM_BINARY" ] || [[ "$RUNTIME_TYPE" == "vm" ]]; then
+		skip "crun-wasm not installed or runtime type is VM"
+	fi
+	cat << EOF > "$CRIO_CONFIG_DIR/99-crun-wasm.conf"
+[crio.runtime]
+default_runtime = "crun-wasm"
+
+[crio.runtime.runtimes.crun-wasm]
+runtime_path = "/usr/bin/crun"
+
+[[crio.runtime.runtimes.crun-wasm.platform_runtime_paths]]
+platform = "wasi/wasm32"
+runtime_path = "/usr/bin/crun-wasm"
```

Comment:
Can we do something like this: 
```
[crio.runtime.runtimes.crun]

platform_runtimes_paths = { "wasi/wasm32":  "/usr/bin/crun-wasm", "abc/def", "/usr/bin/acme" }

Reply:
@mrunalp Are you suggesting that instead of creating a separate table, we should include the `platform_runtime_paths` field in the same RuntimeHandlers table?

Reply:
Yes.